src/modules/candy-machine/transactionBuilders/initCandyMachineBuilder.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  createInitializeCandyMachineInstruction,
+  InitializeCandyMachineInstructionAccounts,
+  InitializeCandyMachineInstructionArgs,
+  PROGRAM_ID as CANDY_MACHINE_PROGRAM_ID,
+} from '@metaplex-foundation/mpl-candy-machine';
+import { Connection, LAMPORTS_PER_SOL } from '@solana/web3.js';
+import { createAccountBuilder } from '../../../programs';
+import { Signer, TransactionBuilder } from '../../../shared';
+import { logDebug } from '../../../utils';
+import { CandyMachineModel } from '../models/CandyMachine';
+import { InitCandyMachineInput } from '../operations';
+
+export type InitCandyMachineBuilderParams = Required<
+  Omit<InitCandyMachineInput, 'confirmOptions'>
+> & { confirmOptions: InitCandyMachineInput['confirmOptions']; connection: Connection };
+
+export async function initCandyMachineBuilder(
+  params: InitCandyMachineBuilderParams
+): Promise<TransactionBuilder> {
+  const { candyMachineModel, candyMachine, wallet, payer, authority, connection } = params;
+  const candyMachineData = candyMachineModel.getData(candyMachine.publicKey);
+
+  const args: InitializeCandyMachineInstructionArgs = { data: candyMachineData };
+  const accounts: InitializeCandyMachineInstructionAccounts = {
+    candyMachine: candyMachine.publicKey,
+    wallet,
+    payer: payer.publicKey,
+    authority,
+  };
+  const initCandyMachineIx = createInitializeCandyMachineInstruction(accounts, args);
+  const createCandyMachine = await createCandyMachineAccountBuilder(
+    payer,
+    candyMachine,
+    candyMachineModel,
+    connection
+  );
+
+  return TransactionBuilder.make()
+    .add(createCandyMachine)
+    .add({
+      instruction: initCandyMachineIx,
+      signers: [payer],
+      key: 'initializeCandyMachine',
+    });
+}
+
+async function createCandyMachineAccountBuilder(
```

Comment:
I actually don't know how I feel about that whole `create` versus `init` now.

I understand that technically, the Candy Machine is first created by the system program and then initialized by the candy machine program but that should be transparent to our users.

We're exposing too low-level language here.